server/logger.js

```diff
@@ -26,12 +30,18 @@ if (dbServer) {
   );
 }
 
-if (process.env.NODE_ENV !== 'test') {
+if (process.env.NODE_ENV === 'local') {
```

Comment:
Why was this check switch?

Reply:
We want to emit JSON logs in OCP so that elastic search can index them. 

While only in local we want to use a more human readable format.

Reply:
Makes sense. 👍